underlay/src/main/java/bio/terra/tanagra/utils/GoogleBigQuery.java

```diff
@@ -437,17 +437,23 @@ static boolean isRetryable(Exception ex) {
     if (!(ex instanceof BigQueryException)) {
       return false;
     }
-    int statusCode = ((BigQueryException) ex).getCode();
+    BigQueryException bqEx = (BigQueryException) ex;
+    int statusCode = bqEx.getCode();
     LOGGER.error("Caught a BQ error (status code = {}).", statusCode, ex);
 
+    if (statusCode == HttpStatus.SC_FORBIDDEN) {
+      return !bqEx.getMessage()
+          .contains(
+              "Response too large to return. Consider specifying a destination table in your job configuration.");
```

Comment:
Is there not a way to check this other than by text?

Reply:
Found a custom property on the BQ excpetion I can check instead.